user/g4music/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "g4music"
+pkgver = "3.8.2"
+pkgrel = 0
+build_style = "meson"
+# Only checks the validity of the desktop files
+configure_args = ["-Dunit_tests=disabled"]
+hostmakedepends = [
+    "meson",
+    "pkgconf"
+]
+makedepends = [
+    "desktop-file-utils",
+    "gettext-devel",
+    "gobject-introspection",
+    "gst-plugins-bad",
+    "gst-plugins-base-devel",
+    "gst-plugins-good",
+    "gst-plugins-ugly",
+    "gstreamer-devel",
+    "gtk4-devel",
+    "libadwaita-devel",
+    "vala-devel"
```

Comment:
this is hostmakedepends (and probably doesn't have to be -devel)

Reply:
i didn't mean every single dependency is hostmakedepends, just the lines mentioned

Reply:
fixed for ya